CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,120 @@
+# Code of Conduct
+
+We follow the [CNCF Code of Conduct][cncf-coc].
+
+As part of our pledge to respect all people, in both live in-person and online
+interactions, we are committed to providing a friendly, safe and welcoming
+environment for all, regardless of gender, gender identity and expression,
+sexual orientation, disability, physical appearance, body size, race, religion,
+native language, operating system choice, current software stack or prior
+experience.
+
+In keeping with this commitment, we offer the following guidelines:
+
+* Welcome newcomers at any stage of expertise.
+  * Everyone has something to contribute.
+  * Everyone deserves access to materials and community that will help them
+       learn.
+  * As long as an individual can be respectful and not disruptive to other
+       participants, they deserve to participate.
+* Provide open and free material.
+* Be kind and courteous.
+  * Interpret the arguments of others in good faith, offering private
+       constructive feedback when communication style bears improvement.
+  * Leave space for quieter voices.
+* Consider who is not in the room.
+  * Invite participation from experts or user community representatives
+       outside of the working group.
+  * Participate in online forums to be inclusive of those who cannot attend
+       meetings.
+* Work performed within this group, either finalized or in draft, is to be
+     used in accordance with the group [Mission and
+     Charter][charter],
+     the open source license, and to be used for the equal benefit of all
+     members of the community.
+
+## Incident handling and escalation
+
+Content for the purposes of the code of conduct as well as incident is defined
+not only as published or draft content but also online discourse, such as slack
+messages or emails, and interactions at in-person events.  If an incident
+involving community conduct occurs, please follow the guidelines below on how to
+handle and report the issue:
+
+* If you see content that clearly does not meet the official Code of Conduct,
+  please send an slack message to the Co-Chair/TL and the creator of the content. (For
+  more details refer to the [CNCF Code of Conduct][cncf-coc]).  If it is
+  regarding a co-chair, reach out to the two other chairs directly if you are
+  uncomfortable using the mailing list.
+* If you are uncomfortable with a piece of content (but it may not necessarily
```

Comment:
Hmm, yes – I can understand that. Perhaps it would be good to explicitly state that you can also turn to someone you know / trust / are close to in order to express concerns?

Reply:
The CNCF has taken a lot of time and trouble writing their COC, so I think it most appropriate we follow what they do already.  They've already spent the time thinking of all the corner cases (or running into them painfully).

Reply:
I don't think that helps either.  What if you are new to the community?  

Here is what the CNCF has on reporting.
https://github.com/cncf/foundation/blob/main/code-of-conduct.md#reporting

Reply:
It doesn't force anyone, it encourages people to speak up if they see something they don't like. What should be normal and is in no way outstanding. 

The CNCF CoC addresses this by going to a top-level escalation, rather than just simply solving this between people.